src/helpers/map.js

```diff
@@ -198,9 +138,38 @@ export function getTraversableCells(cpt, map, me) {
   for (x = 0; x < xl; x++) {
     for (let y = 0; y < yl; y++) {
       const objRadius = 20; // TODO: Set radius to be correct value for each cell object
-      fillGridWithSubgrid(emptyCells, traversableCellsInTile(isTraversable(map[x][y], me),
+      fillGridWithSubgrid(emptyCells, traversableCellsInTile(isTraversable(map[x][y]),
         cpt, objRadius), x * cpt, y * cpt);
     }
   }
   return emptyCells;
 }
+
+/*
+ * Returns the position (in pixels x,y and grid positions xg, yg
+ * of first of the specified tile type to appear starting in the
+ * top left corner and moving in a page-reading fashion.
+ */
+export function findTile(targetTile) {
```

Comment:
Can you add to the comment what we use this function for? I assume it's for finding where something (like the enemy flag) is. Something there's only one of that we want to seek toward.